custom_components/intesisbox/intesisbox.py

```diff
@@ -111,8 +134,9 @@ def data_received(self, data):
                 if cmd == "ID":
                     self._parse_id_received(args)
                     self._connectionStatus = API_AUTHENTICATED
-                    _ = asyncio.ensure_future(self.keep_alive())
+                    # _ = asyncio.ensure_future(self.keep_alive())
```

Comment:
Please remove the old commented code (here and in a few other places)

Reply:
Noted. Shall I delete and re-do it?